atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -122,6 +125,12 @@
             = "/" + CLIENT + "/timestamp-management/fast-forward?currentTimestamp=" + EMBEDDED_BOUND;
     private static final MappingBuilder TIMELOCK_FF_MAPPING = post(urlEqualTo(TIMELOCK_FF_PATH));
 
+    private static final ScheduledExecutorService EXECUTOR = Executors.newSingleThreadScheduledExecutor(
+            new ThreadFactoryBuilder()
+                    .setNameFormat(TransactionManagersTest.class.getSimpleName() + "-%d")
+                    .setDaemon(true)
+                    .build());
+
```

Comment:
Should make this a memorized supplier so we don’t create it on class load, and we would still need to shut it down if we create it here.